test_df/translate_sequences_script.py

```diff
@@ -7,13 +7,35 @@
 from df.bio_helper import column_to_sequences, sequences_to_column
```

Comment:
This file duplicates the script found in the Data Function .yaml file.   (Just making sure I've done this correctly.)

Reply:
You might want to look at what Dave Cosgrove has done for the some of his chemistry data functions.  He wrote `test_chem/yaml_to_script.py` which has code to copy the script from DataFxns (assuming that DataFxns and DataFxnPylibTest are in the same directory). 

See `test_chem.test_ArbitraryReaction.ScriptTest.setUp()`

Reply:
I used `test_chem/yaml_to_script.py` and verified the content of the `execute` function here matches the results of that script as well as that in the .yaml file. (One vestigial import which came from the original example was removed.)   This test is slightly different than those in `test_chem/`, however, as there is a controlling script, `test_data_functions.py`, which imports the `execute` function from this file.  Both of these files pre-existed in this arrangement, and I modified the content of each to support the new start site detection functionality.  All new tests that I've added as well as the original tests pass.